From ee2037592330754bee027b1cccdf1c46bdb7309b Mon Sep 17 00:00:00 2001
From: Sergei Trofimovich <slyich@gmail.com>
Date: Thu, 18 Aug 2022 11:45:06 +0100
Subject: [PATCH] Fix --enable-configure-args description.

Before the change the description said:

    Handle all compiler warnings as errors

Looks like a leftover from --enable-werror copy.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index a987d27e4..a6433faa8 100644
--- a/configure.ac
+++ b/configure.ac
@@ -574,7 +574,7 @@ AM_CONDITIONAL([HAVE_GMODULE], [test -n "$g_module_supported" && \
                                 test x"$textmode_x11_support" = x"yes" -o x"$enable_aspell" = x"yes"])
 
 AC_ARG_ENABLE([configure-args],
-    AS_HELP_STRING([--enable-configure-args], [Handle all compiler warnings as errors]))
+    AS_HELP_STRING([--enable-configure-args], [Embed ./configure arguments into binaries]))
 if test "x$enable_configure_args" != xno; then
     AC_DEFINE([ENABLE_CONFIGURE_ARGS], 1, [Define to enable showing configure arguments in help])
     AC_DEFINE_UNQUOTED([MC_CONFIGURE_ARGS], ["$ac_configure_args"], [MC configure arguments])
-- 
2.37.1

